datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -293,6 +300,9 @@ private static void configureDatasetResolvers(final RuntimeWiring.Builder builde
                                     UPSTREAM_LINEAGE_TYPE,
                                     (env) -> ((Entity) env.getSource()).getUrn()))
                     )
+                    .dataFetcher("schemaMetadata", new AuthenticatedResolver<>(
```

Comment:
So I guess for every aspect we want to support versioned aspect fetching on we'll need this.... Okay

Reply:
For now, yes. However, when we autogenerate the graph we can also autogenerate these.